src/database/migrations/0055-alter-base_fields-add-relevance_duration.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE base_fields
+ADD COLUMN relevance_duration integer;
```

Comment:
This means the maximum duration we support is under 69 years. I'm trying to think if we can reasonably expect a duration of, say, a century.

How about the current president of an organization? We would not expect that to last more than about 100 years, right?

I wish we could use a more explicit duration type such as postgres' `interval` here and TypeScript's `TimeDuration` elsewhere. That might add some complexity, though.

What would it mean to have a negative duration here? Is it allowed?

Reply:
This is a good point and I wonder if we should just make it days.  I'm trying to think of a use case that would be in terms of hours.  It seems more likely that something would be 100 years than need the granularity of seconds, hours, or minutes.

Reply:
UPDATE: spoke with @bickelj and we landed on hours as a nice middle ground here!